cli/commands/run/flags.go

```diff
@@ -99,8 +99,11 @@ const (
 	DeprecatedSourceUpdateFlagName                   = "source-update"
 	DeprecatedSourceFlagName                         = "source"
 	DeprecatedNoAutoApproveFlagName                  = "no-auto-approve"
+	DeprecatedAutoApproveFlagName                    = "auto-approve"
```

Comment:
If a user runs `terragrunt apply -auto-approve`, will this fire on that?

Reply:
> If a user runs `terragrunt apply -auto-approve`, will this fire on that?

Yes, it will work only for the `TERRAGRUNT_AUTO_APPROVE` env vars.

https://github.com/gruntwork-io/terragrunt/blob/e42f9ef22a63c65cda80b5dbeaeb77f828e46f25/cli/commands/run/flags.go#L185